src/commands/codepush/patch.ts

```diff
@@ -0,0 +1,102 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp, shortName, longName, required, hasArg, position, name } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest, clientCall } from "../../util/apis";
+import * as _ from "lodash";
```

Comment:
Unused import here, need to remove `import * as _ from "lodash";`

Reply:
Fixed.